bo/messlokation.go

```diff
@@ -13,15 +14,17 @@ import (
-	Messlokationszaehler         []Zaehler           `json:"messlokationszaehler,omitempty"`                                                                            // Messlokationszaehler meters associated to this Messlokation
+	MesslokationsId                 string                     `json:"messlokationsId,omitempty" example:"DE0123456789012345678901234567890" validate:"alphanum,required,len=33"` // MesslokationsId is the ID of the metering location
+	Sparte                          sparte.Sparte              `json:"sparte,omitempty" validate:"required"`                                                                      // Sparte is the division
+	NetzebeneMessung                netzebene.Netzebene        `json:"netzebeneMessung,omitempty"`                                                                                // NetzebeneMessung is the grid level of measurement
+	MessgebietNr                    string                     `json:"messgebietNr,omitempty"`                                                                                    // MessgebietNr is the number of the measurement area in ene't database
+	Geraete                         []com.Hardware             `json:"geraete,omitempty"`                                                                                         // Geraete is a list of devices
+	Messdienstleistung              *com.Dienstleistung        `json:"messdienstleistung,omitempty"`                                                                              // Messdienstleistung is a metering services
+	GrundzustaendigerMsbCodeNr      string                     `json:"grundzustaendigerMSBCodeNr,omitempty" validate:"omitempty,numeric,len=13"`                                  // GrundzustaendigerMsbCodeNr is the code number of the "grundzuständige Messstellenbetreiber", responsitble for this MeLo
+	GrundzustaendigerMsbImCodeNr    string                     `json:"GrundzustaendigerMsbImCodeNr,omitempty" validate:"omitempty,numeric,len=13"`                                // GrundzustaendigerMsbImCodeNr si the code number of the "grundzuständige Messsstellenbetreiber", responsible for intelligent meters at this MeLo
+	Messlokationszaehler            []Zaehler                  `json:"messlokationszaehler,omitempty"`                                                                            // Messlokationszaehler meters associated to this Messlokation
+	Gasqualitaet                    *gasqualitaet.Gasqualitaet `json:"gasqualitaet"`                                                                                              // gasqualitaet für EDIFACT mapping
+	Abrechnungmessstellenbetriebnna *bool                      `json:"abrechnungmessstellenbetriebnna"`                                                                           //Dieser Wert ist true, falls die Abrechnungs des Messstellenbetriebs die Netznutzungsabrechnung enthält. false andernfalls
```

Comment:
at all other attributes there is `omitempty` present. 
I am not totally sure what it does, but it's odd that the new ones do not have it.